src/main/java/com/coveros/selenified/Browser.java

```diff
@@ -103,4 +134,58 @@ public static BrowserName lookup(String b) throws InvalidBrowserException {
         }
         throw new InvalidBrowserException("The selected browser " + b + " is not an applicable choice");
     }
+
+    /**
+     * sets the browser details (name, version, device, orientation, os) into
+     * the device capabilities
+     *
+     * @param capabilities - the DesiredCapabilities object, which will get set with all browser capabilities
+     */
+    public DesiredCapabilities setBrowserCapabilities(DesiredCapabilities capabilities) {
+        // null check
+        if (capabilities == null) {
+            capabilities = new DesiredCapabilities();
+        }
+        // determine the browser information
+        if (capabilities.getBrowserName() == null || "".equals(capabilities.getBrowserName())) {
+            capabilities.setCapability(CapabilityType.BROWSER_NAME, getName().toString().toLowerCase());
+        }
+        if (getVersion() != null) {
+            capabilities.setCapability(CapabilityType.VERSION, getVersion());
+        }
+        if (getPlatform() != null) {
+            capabilities.setCapability(CapabilityType.PLATFORM, getPlatform());
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "PLATFORM"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)